### PR TITLE
fix: Modify the documentation of the Demos and Tutorials chapter (#38054)

### PR DESCRIPTION
**Pull request by HuaSheng2000**
issue: #38054 
- Delete the table about tutorials in the English document, use link to replace the existing table, and add corresponding descriptions (the table content is outdated, and using link can reduce maintenance costs).
- Add link and corresponding descriptions of tutorials in Chinese documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,25 +89,9 @@ Milvus is designed to handle vector search at scale. Users can store vectors, wh
...
-| [Recommender System](recommendation_system.md) | Recommendation System | vector search |
-| [Video Similarity Search](video_similarity_search.md) | Semantic Search | vector search |
-| [Audio Similarity Search](audio_similarity_search.md) | Semantic Search | vector search |
-| [DNA Classification](dna_sequence_classification.md) | Classification | vector search |
-| [Graph RAG with Milvus](graph_rag_with_milvus.md) | RAG | vector search |
-| [Contextual Retrieval with Milvus](contextual_retrieval_with_milvus.md) | RAG, Semantic Search | vector search |
-| [HDBSCAN Clustering with Milvus](hdbscan_clustering_with_milvus.md) | Clustering | vector search |
-| [Use ColPali for Multi-Modal Retrieval with Milvus](use_ColPali_with_milvus.md) | RAG, Semantic Search | vector search |
-| [Vector Visualization](vector_visualization.md) | Data Visualization | vector search |
+Here is a selection of demos and tutorials to show how to build various types of AI applications made with Milvus:
```
Comment (yanliang567) [inline README.md:94]: shall we not just remove the list of demos and tutorials, but correct the link to each demo and tutorial?

Comment (HuaSheng2000) [reply README.md:94]: I updated the table and tutorial links based on [this link](https://raw.githubusercontent.com/milvus-io/milvus-docs/refs/heads/v2.5.x/site/en/tutorials/tutorials-overview.md), and added the table for the Chinese documentation.(all links available)

**Pull request merged**